src/solver/implicitgemm_util.hpp

```diff
@@ -18,14 +18,9 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_BLOCK_SYNC_LDS_WITHOUT_SY
 #define WORKAROUND_SWDEV_229564 1
 // workaround for buffer load/store fp16/bfp16 intrinsic bug
 #define WORKAROUND_SWDEV_231101 1
-// LLVM xdlops instrinsic will do unnecessey VGRP <--> AGPR movement, and result in
-// register spill, for bfloat16 datatype, when doing wave-wise GEMM larger than 64x64
-#define WORKAROUND_SWDEV_240356 1
```

Comment:
Why this workaround removed? Even if recent ROCm have been fixed, older ROCm still have issue. So latest MIOpen installed on existing old ROCm may have issues.

Reply:
We fixed this issue by fixing our kernel implementation.